src/fileformats/ocd_file_import.h

```diff
@@ -362,6 +377,11 @@ class OcdFileImport : public Importer
 	int ocd_version;
 };
 
+inline
```

Comment:
Just move it into the cpp file. It is used only there, and it will still be inlined based on compiler decisions.

(Extended comment: The whole class declaration could be moved to the cpp file, but this would prevent unit testing. The whole class could be a separate translation unit, and then the `atEnd` definition would indeed need go to the header to allow inlining. Today's preferred form is to put the body into the class definition instead of an explicit `inline` - again in order to let the compile decide about inlining.)